### PR TITLE
chore: remove all the remnants of module.id

### PR DESCRIPTION
**Pull request by Foxandxss**
the `module.id` on the lib existed for the sole purpose of the demo. Now that we have that awesome loader, we don't need to do that anymore.

For the sake of convenience, I added it to the integration tests as well.

### CODE REVIEW

File: src/lib/systemjs-angular-loader.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+var templateUrlRegex = /templateUrl\s*:(\s*['"`](.*?)['"`]\s*)/gm;
```
Comment (filipesilva) [inline src/lib/systemjs-angular-loader.js]: Is this file necessary? The library shouldn't need systemjs config.

Comment (Foxandxss) [reply src/lib/systemjs-angular-loader.js]: it is needed for testing. Karma uses systemjs to run the tests so it needs either `module.id` or the loader.

I tried to use the loader at `demo/` but I couldn't. It needed to be on SystemJS baseURL :/

Comment (filipesilva) [reply src/lib/systemjs-angular-loader.js]: Can't it be inlined in the karma configuration like the rest of the current unit testing configs?

Comment (Foxandxss) [reply src/lib/systemjs-angular-loader.js]: Totally

Comment (filipesilva) [reply src/lib/systemjs-angular-loader.js]: I'd prefer that if you don't mind. It feels weird to have it here, and in general the library source shouldn't have to care about systemjs.
